migrating_from_ocp_3_to_4/index.adoc

```diff
@@ -0,0 +1,86 @@
+* xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migration-state-migration-cli_advanced-migration-options-3-4[Running a state migration]
+* xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migration-hooks_advanced-migration-options-3-4[Creating migration hooks]
+* xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migration-plan-options_advanced-migration-options-3-4[Editing, excluding, and mapping migrated resources]
+* xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migration-controller-options_advanced-migration-options-3-4[Configuring the migration controller for large migrations]
+
+[id="mtc-overview-troubleshooting-mtc"]
+== Troubleshooting migrations
+You can perform the following troubleshooting tasks:
+
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-viewing-migration-plan-resources_troubleshooting-3-4[Viewing plan resources]
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-viewing-migration-plan-log_troubleshooting-3-4[Viewing the migration plan aggregated log file]
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-using-mig-log-reader_troubleshooting-3-4[Using the migration log reader]
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-accessing-performance-metrics_troubleshooting-3-4[Accessing performance metrics]
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-using-must-gather_troubleshooting-3-4[Using the `must-gather` tool]
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-debugging-velero-resources_troubleshooting-3-4[Using the Velero CLI to debug Backup and Restore CRs]
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-partial-failure-velero_troubleshooting-3-4[Debugging a partial migration failure]
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-using-mtc-crs-for-troubleshooting_troubleshooting-3-4[Using MTC custom resources for troubleshooting]
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#common-issues-and-concerns_troubleshooting-3-4[Checking common issues and concerns]
+
+[id="mtc-overview-roll-back-mtc"]
+== Rolling back a migration
+You can xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#rolling-back-migration_troubleshooting-3-4[roll back a migration] by using the MTC web console, the CLI or manually.
+
+[id="mtc-overview-uninstall-mtc"]
+== Uninstalling MTC and deleting resources
+You can uninstall the MTC and delete its resources to clean up the cluster. To learn more see xref:../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster-using-web-console_olm-deleting-operators-from-cluster[deleting Operators from a cluster using the web console].
```

Comment:
THis is not correct. We have a section about [uninstalling the Operator and cleaning resources](https://deploy-preview-40171--osdocs.netlify.app/openshift-enterprise/latest/migration_toolkit_for_containers/troubleshooting-mtc.html#migration-uninstalling-mtc-clean-up_troubleshooting-mtc). You need to link to that section.

I moved it [recently](https://github.com/openshift/openshift-docs/pull/40633), so rebase your branch so that you can link correctly.

Reply:
acked